quickstart/ia32-generic-qemu.md

```diff
@@ -148,6 +148,57 @@ There are few steps to follow:
       sudo virsh net-start --network default
       ```
 
+    - If IPv6 is needed change the configuration of `virbr0`
+
+      ```bash
+      sudo virsh net-destroy default
+      sudo virsh net-edit default
+      ```
+
+      The commands above open the editor of the configuration file of `virbr0`. The are two necessary changes:
+
+      - Add IPv6 address to the bridge interface:
+
+        ```xml
+        <ip family='ipv6' address='2001:db8:dead:beef:fe::2' prefix='64'>
+        </ip>
+        ```
+
+      - Enable NAT for IPv6:
+
+        ```xml
+        <forward mode='nat'>
+          <nat ipv6='yes'/>
+        </forward>
+        ```
+
+      The overall config should look something like this:
+
+      ```xml
+      <network ipv6='yes'>
```

Comment:
this ipv6='yes' it's not described above and not sure if required.

Reply:
yep, not needed